src/Symfony/Component/Dotenv/Dotenv.php

```diff
@@ -456,6 +457,14 @@ private function resolveVariables($value)
                 $value = (string) getenv($name);
             }
 
+            if ('' === $value && isset($matches['default_value'])) {
```

Comment:
we should also check that there is both an opening and closing bracket
and when on is missing, we should append the content of the default_value index as a regular string
please add a test case covering this situation.

Reply:
@nicolas-grekas If you are talking about the following case:
`BAR=${FOO:-TEST`, it is already handled with a `FormatException : Unclosed braces on variable expansion`.

Reply:
yes, thanks for the precision
one more question: can we enter this "if" when there's no opening bracket? I feel like yes, while this should fail, isn't it?

Reply:
You are right, I added a check for a missing opening brace and the following case:
```
BAR=$FOO:-a
```
now throws a FormatException with `Variable expansion "$FOO:-a" should be wrapped in braces`